node/src/chain_spec.rs

```diff
@@ -72,6 +73,20 @@ pub fn development_config() -> Result<ChainSpec, String> {
 				"0xec00ad0ec6eeb271a9689888f644d9262016a26a25314ff4ff5d756404c44112"
 					.parse()
 					.unwrap(),
+				// SnowBridged Assets
+				vec![
```

Comment:
I can see the config for `bridged_assets` is defined, but the corresponding `asset` is not created. Am I missing the config for the assets pallet?

Reply:
The asset is being initialized in add_extra_genesis in pallests/bridged_asset.
This is because the bridged_asset pallet is tightly-coupled ("pub trait Trait: assets::Trait") to the assets pallet, it seemed natural to have the bridged_assets pallet call asset_issue from the assets pallet.

Reply:
Good approach, keep in mind the bond between the integer balance and smallest denomination balance

Reply:
As per my understanding, all input, output, processing, computation and storage are done in the "smallest denomination balance", while the representation of these values is left to the ui/fe using the decimal information.
Am I right?
If yes, then I can just use the smallest denomination balance everywhere.

Reply:
Yes, you are right, no operations on the node side should involve the decimal information.